src/functionalTest/resources/templates/aip-minimal-appeal-submitted.json

```diff
@@ -12,7 +12,7 @@
     {
       "id": "1",
       "value": {
-        "Iceland": "IS"
+        "code": "IS"
```

Comment:
This is already implemented in RIA-3628, which will be merged to master. Rebase on master should cover this

Reply:
RIA-3628 does not have any changes in this service. You are probably confused of changes in case-api.

Reply:
Got it, thanks